source/index.txt

```diff
@@ -68,6 +68,17 @@ binaries:
 Versioning
 ----------
 
+.. important::
+
+   It is recommended you always use the latest release version of the 
+   {+dbtools+}. To check your current version of the {+dbtools+}, run the 
```

Comment:
**Nit**

Small nit, Im actually not sure which one sounds better feel free to disregard. 

```suggestion
   {+dbtools+}. To check the version of the {+dbtools+}, run the 
```